### DISCUSSION TITLE
Wifi gateway

### DISCUSSION THREAD

**Discussion by notworken** [spbrogan/sensor.mopeka_pro_check#9] [Ideas]
Do you have any idea how hard it would be to make this work with the Mopeka 024-3000 Tank Check Bluetooth Gateway/WiFi Bridge? Or maybe just setup a RPi is do that function? My HA system isn't close enough for direct bluetooth.

**Comment by spbrogan** [comment]
I haven't looked too much a wifi bridge/gateway but i think working with that you would need to do some sort of "cloud polling" solution. 

For this issue i have mostly switched to using esphome.  If you are not familiar it is a super cheap and easy way to use an esp32 and connect it to HA.  

I have a custom component that is waiting to get merged into mainline to do it.  
See here: https://github.com/spbrogan/esphome/tree/mopeka_dev

you can see how to easily do this in a discussion i had with another person.  https://github.com/spbrogan/sensor.mopeka_pro_check/discussions/7
